### PR TITLE
Support linting of individual files via command-line

### PR DESCRIPTION
**Pull request by rcatlin**
```
ryan 福 ~/Projects/chassis/chassis ➤ 89c306c|feature/lint-individual-files⚡
8699 ± : python tools/lint.py tools/lint.py                                       [11d1h45m] ✹
No config file found, using default configuration
(env)
ryan 福 ~/Projects/chassis/chassis ➤ 89c306c|feature/lint-individual-files⚡
```

:horse: Yee haw! :cactus:

### CODE REVIEW

**Comment by rcatlin** [comment]
@mattmillr review?

File: chassis/tools/lint.py [hunk 1]
```diff
@@ -11,25 +11,36 @@
...
+    'too-many-public-methods',
+    # 'import-error',
     'fixme',
-    # 'file-ignored'
+    'too-many-lines',
+    'cyclic-import',
+    'duplicate-code',
+    'file-ignored',
+    'interface-not-implemented',
+    'abstract-class-not-used'
```
Comment (mattmillr) [inline chassis/tools/lint.py]: Looks like you're adding back a bunch of supressions we got rid of in Chassis :frowning:

Comment (rcatlin) [inline chassis/tools/lint.py]: oh yikes, yes. I'll remove.

**Comment by mattmillr** [comment]
:+1: plz merge sir :smile_cat:

**Pull request merged**